### DISCUSSION TITLE
Idea: Eaglercraft

### DISCUSSION THREAD

**Discussion by FusionCow** [DustinBrett/daedalOS#338] [Feature Requests]
Hello,

just a simple idea I had, basically, eaglercraft is a cracked minecraft client written in js, and I thought it would be a cool application to throw in.

Thanks

**Comment by DustinBrett** [comment]
> Hello,
> 
> just a simple idea I had, basically, eaglercraft is a cracked minecraft client written in js, and I thought it would be a cool application to throw in.
> 
> Thanks

Hi @FusionCow,

Thanks for the suggestion. I will keep this in mind for the future. Currently I have a Minecraft like game called [ClassiCube](https://www.classicube.net/). I see Eaglercraft at https://minecraftforfreex.com/eaglercraft/ and it doesn't look simple to integrate.

**Reply by FusionCow** [reply]
Sorry for the bother, but I also want to be able to integrate my own js apps, how do you register an app into your code? Like if I just wanted to get a simple iframe html app, how would I go about that

**Reply by Pixel556** [reply]
ClassiCube is broken, I ran it once and after that it stopped working.

**Reply by DustinBrett** [reply]
There is a known issue where 2nd runs of a few apps fail because of the way the browser env is initialized. Thanks for the report.

https://github.com/DustinBrett/daedalOS/issues/250

**Comment by DustinBrett** [comment]
> Sorry for the bother, but I also want to be able to integrate my own js apps, how do you register an app into your code? Like if I just wanted to get a simple iframe html app, how would I go about that

@FusionCow I do not yet have third party app support, but I will eventually be adding it.

If you want to fork the code and make an app, you can see many examples in `components/apps` of how to make apps. Some iframe apps are `Paint` & `IRC`. But my preference is for non-iframe apps, which most others are.

**Reply by FusionCow** [reply]
Thank you so much.
